core/src/main/scala/org/scalafmt/util/TokenOps.scala

```diff
@@ -221,7 +221,11 @@ object TokenOps {
   )
 
   def isSymbolicIdent(tok: Token): Boolean = tok match {
```

Comment:
The name of this function is confusing (or maybe I'm missing something). `isSymbolicIdent`? As in "is this a symbolic identifier"? Shouldn't this be `isOperator`?